sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -510,10 +510,12 @@ class Analyzer(
       // collect all the found AggregateExpression, so we can check an expression is part of
       // any AggregateExpression or not.
       val aggsBuffer = ArrayBuffer[Expression]()
+
```

Comment:
nit: plz revert the unencessary changes.

Reply:
> nit: plz revert the unencessary changes. (Unrelated changes might lead to revert/backport failures sometimes...)

Sorry..., forgot to check diff.